src/main/resources/com/google/api/codegen/metadatagen/py/setup.py.snip

```diff
@@ -0,0 +1,50 @@
+@snippet generateFilename()
+    setup.py
+@end
+
+@snippet generateBody(model)
+    """A setup module for the GRPC google-{@context.getShortName} service.
+
+    See:
+    https://packaging.python.org/en/latest/distributing.html
+    https://github.com/pypa/sampleproject
+    """
+
+    import setuptools
+
+    from setuptools import setup, find_packages
+
+    install_requires = [
+      'oauth2client>={@context.getDependencyValue("auth:python:version")}, <{@context.getDependencyValue("auth:python:next_version")}',
```

Comment:
I feel like it would be more natural to use a dotted path instead of a coloned path.